kafka-queues/pom.xml

```
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>service</artifactId>
+        <groupId>uk.gov.gchq.palisade</groupId>
+        <version>0.4.0-${revision}</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <artifactId>kafka-queues</artifactId>
```

update the name to be a bit more descriptive
```suggestion
    <artifactId>create-kafka-queues</artifactId>
```